test/src/test/java/jenkins/security/BasicHeaderProcessorTest.java

```diff
@@ -111,28 +94,53 @@ public void testAuthHeaderCaseInSensitive() throws Exception {
         wc = j.createWebClient();
 
         String[] basicCandidates = {"Basic", "BASIC", "basic", "bASIC"};
-        
+
         for (String prefix : basicCandidates) {
             // call with API token
             ApiTokenProperty t = foo.getProperty(ApiTokenProperty.class);
             final String token = t.getApiToken();
-            String authCode1 = encrypt(prefix,"foo:"+token);
+            String authCode1 = encode(prefix,"foo:"+token);
             makeRequestWithAuthCodeAndVerify(authCode1, "foo");
-            
+
             // call with invalid API token
-            String authCode2 = encrypt(prefix,"foo:abcd"+token);
+            String authCode2 = encode(prefix,"foo:abcd"+token);
             makeRequestWithAuthCodeAndFail(authCode2);
 
             // call with password
-            String authCode3 = encrypt(prefix,"foo:foo");
+            String authCode3 = encode(prefix,"foo:foo");
             makeRequestWithAuthCodeAndVerify(authCode3, "foo");
 
             // call with incorrect password
-            String authCode4 = encrypt(prefix,"foo:bar");
+            String authCode4 = encode(prefix,"foo:bar");
             makeRequestWithAuthCodeAndFail(authCode4);
         }
     }
 
+    private String encode(String prefix, String userAndPass) {
+        if (userAndPass==null) {
+            return null;
+        }
+        return prefix+" "+Scrambler.scramble(userAndPass);
```

Comment:
Use `Base64`. `Scrambler` should be considered deprecated.

Reply:
As you showed me the existence of `java.util.Base64` in Java 8, I think you are talking about that one.